game.go

```diff
@@ -310,7 +314,8 @@ type cameraConfig struct {
 }
 
 type mapConfig struct {
-	Mode string `json:"mode"`
+	Mode      string `json:"mode"`
+	Resizable bool   `json:"resizable"`
```

Comment:
resizable is a window property, not a map property.

Reply:
fixed